posthog-core/src/index.ts

```diff
@@ -436,6 +502,15 @@ export abstract class PostHogCore {
     return this.setPersistedProperty(PostHogPersistedProperty.OverrideFeatureFlags, flags)
   }
 
+  _sendFeatureFlags(event: string, properties?: { [key: string]: any }): void {
+    // Used for posthog-node only
```

Comment:
If this is node only, why do we want it in the core? Why not have this called by the already separate capture call?

Reply:
because it meant exposing the `enqueue` internals of core, which felt a bit more messy to me.

It's not that other libraries can't make use of this, it's just that _today_ it's used for node only. Hmm, actually, comment is confusing, will remove it.